tests/e2e/helpers/tools/tools.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tools
+
+import (
+	"crypto/tls"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+)
+
+func GetFileFromURL(url string, fileName string, skipVerify bool) error {
+	if !skipVerify {
+		http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+	}
+
+	data, err := http.Get(url)
+	if err != nil {
+		return err
+	}
+	defer data.Body.Close()
+
+	// Create file
+	file, err := os.Create(fileName)
+	if err != nil {
+		return err
+	}
+
+	// Save data in file
+	_, err = io.Copy(file, data.Body)
+	return err
+}
+
+func GetFiles(dir string, pattern string) ([]string, error) {
```

Comment:
this file looks a good candidate that could go into `ele-testhelpers`

Reply:
Yes, can I do it in another PR? As I will have to do 2 PR in 2 differents repos.

Reply:
yup definetly, this is just a suggestion, makes completely sense having a follow-up for it

Reply:
OK, I will do it quickly.

Reply:
Done here: https://github.com/rancher-sandbox/os2/pull/102